src/gwt/www/templates/encrypted-sign-in.htm

```diff
@@ -201,8 +201,11 @@ <h1 id="caption">Sign in to RStudio</h1>
                 autocomplete='off' /><br />
       </p>
       <p style="display: #staySignedInDisplay#;">
+         <p style="display: #authTimeoutMinutesDisplay#";">
+            <label style="font-size:10px">You will automatically be signed out after #authTimeoutMinutes# minutes of inactivity.</label>
```

Comment:
Nit: a `label` should always have a `for` attribute (it's a label for a form control). I think this should be a `div` or `span`.